cleanlab/count.py

```diff
@@ -150,9 +154,9 @@ def num_label_issues(
         label_issues_mask = np.zeros(len(labels), dtype=bool)
         label_issues_mask[cl_error_indices] = True
 
-        # Remove label issues if given label == model prediction
-        pred = pred_probs.argmax(axis=1)
-        label_issues_mask[pred == labels] = False
+        # Remove label issues if model prediction is close to given label
+        mask = _reduce_issues(pred_probs=pred_probs, labels=labels)
+        label_issues_mask[mask] = False
```

Comment:
Since the `label_issues_mask` is initialized with `False`s, you only need to reset to `False` at the relevant `cl_error_indices`.

```suggestion
        mask = _reduce_issues(pred_probs=pred_probs[cl_error_indices], labels=labels[cl_error_indices])
        label_issues_mask[cl_error_indices[mask]] = False
```